ndbench-dyno-plugins/src/main/java/com/netflix/ndbench/plugin/dyno/DynoJedisUtils.java

```diff
@@ -15,22 +15,16 @@
  */
 package com.netflix.ndbench.plugin.dyno;
 
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Random;
-import java.util.Set;
-import java.util.concurrent.atomic.AtomicReference;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.netflix.dyno.jedis.DynoJedisClient;
 import com.netflix.dyno.jedis.DynoJedisPipeline;
 import com.netflix.ndbench.api.plugin.DataGenerator;
-
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import redis.clients.jedis.Response;
 
+import java.util.*;
+import java.util.concurrent.atomic.AtomicReference;
```

Comment:
Since you bring `java.util.*;` why do you bring the sublass as well `java.util.concurrent.atomic.AtomicReference;`?